File: sol/solution/4A/Solution.cs
using System;
class Program
{
    static void Main()
    {
        int w = int.Parse(Console.ReadLine());
        if (w > 2 && w % 2 == 0)
        {
            Console.WriteLine("YES");
        }
        else
        {
            Console.WriteLine("NO");
        }
    }
}

File: sol/solution/4A/Solution.rs
use std::io;
fn main() {
    let mut s = String::new();
    io::stdin().read_line(&mut s).unwrap();
    let w: i32 = s.trim().parse().unwrap();
    if w > 2 && w % 2 == 0 {
        println!("YES");
    } else {
        println!("NO");
    }
}